scripts/remove

```diff
@@ -31,6 +32,9 @@ ynh_script_progression --message="Removing Lychee main directory..." --weight=3
 # Remove the app directory securely
 ynh_secure_remove --file="$final_path"
 
+# Remove data directory securely
+ynh_secure_remove --file="$public_path"
```

Comment:
Eeeeh do we want to do this ?

I don't know how this is handled for other app... but in the case of failed-restore-during-failed-backup (for example of nextcloud) it's pretty important to not purge the data ... I havent look what this app does though so I don't know if it's maybe not important ... And overall I don't know if we have a clear recommendation for this

Reply:
Ah ah by the way, we also cleaned and tidied up your data storage 😅.
 I also have to find a way to upgrade from master where Lychee stores the data in FINALPATH to this version where the data is stored  in this folder `/home/yunohost.app/$app`